libs/application/templates/children-residence-change/src/lib/messages/signatureModal.ts

```diff
@@ -0,0 +1,45 @@
+import { defineMessages } from 'react-intl'
+
+// Signature
+export const signatureModal = {
+  general: defineMessages({
+    title: {
+      id: 'crc.application:section.contract.signatureModal.title',
```

Comment:
Wondering about the id's in this file. Normally we have been following the rule `crc.application:section.{constName}.{messagesDefiniationName}.{variableName}` I am wondering in this scope if it makes sense to have just: `crc.application:signatureModal.title`?

I know that this is used in the contract section but just thinking if it makes sense to have as a part of the key here, since maybe we would like to use this modal elsewhere?